src/components/ProductTrainingContext/TOOLTIPS.ts

```diff
@@ -27,6 +28,7 @@ type TooltipData = {
     name: ProductTrainingTooltipName;
     priority: number;
     shouldShow: (props: ShouldShowConditionProps) => boolean;
+    isModalTooltip: boolean;
```

Comment:
Should we just make this have a default or `?`? It feels weird that every modal needs to add this flag